hypha/apply/funds/models/__init__.py

```diff
@@ -14,15 +14,16 @@
 from .submissions import ApplicationRevision, ApplicationSubmission, AssignedReviewers
 
 __all__ = [
-    'ApplicationSubmission',
+    'ApplicationForm',
+    'ApplicationRevision',
     'ApplicationSettings',
+    'ApplicationSubmission',
     'AssignedReviewers',
-    'ApplicationRevision',
-    'ApplicationForm',
-    'ScreeningStatus',
-    'ReviewerRole',
     'Reminder',
+    'ReviewerRole',
     'ReviewerSettings',
+    'RoundsAndLabs',
+    'ScreeningStatus',
```

Comment:
Here I'm just adding the imported `RoundsAndLabs` which was not included in the `__all__`.